apps/editing-toolkit/editing-toolkit-plugin/common/index.php

```diff
@@ -140,7 +140,13 @@ function enqueue_script_and_style() {
  * @return array Gutenberg editor settings with line-height setting always enabled.
  **/
 function wpcom_gutenberg_enable_custom_line_height( $settings ) {
-	$settings['enableCustomLineHeight'] = true;
+	if ( isset( $settings['__experimentalFeatures']['global']['typography'] ) ) {
```

Comment:
Hi, I've already [commented](https://github.com/WordPress/gutenberg/pull/25738#issuecomment-717851579) that this is a bug we need to fix, see https://github.com/WordPress/gutenberg/issues/26537

I was thinking of a way to do it that's more robust than using `__experimental.global.typography.customLineHeight` (the shape of this can change as it's still experimental). What if you hook this function to `after_setup_theme` and make it declare theme support like `add_theme_support('custom-line-height')`?

Reply:
Made it more robust here: https://github.com/Automattic/wp-calypso/pull/46873. Thanks!